usecases/MinimumWorkingExample/mesh/cylinder.xdmf

```diff
@@ -0,0 +1 @@
+<Xdmf Version="3.0"><Domain><Grid Name="Grid"><Geometry GeometryType="XYZ"><DataItem DataType="Float" Dimensions="158 3" Format="HDF" Precision="8">cylinder.h5:/data0</DataItem></Geometry><Topology TopologyType="Tetrahedron" NumberOfElements="428" NodesPerElement="4"><DataItem DataType="Int" Dimensions="428 4" Format="HDF" Precision="8">cylinder.h5:/data1</DataItem></Topology><Attribute Name="gmsh:dim_tags" AttributeType="Vector" Center="Node"><DataItem DataType="Int" Dimensions="158 2" Format="HDF" Precision="8">cylinder.h5:/data2</DataItem></Attribute><Attribute Name="gmsh:physical" AttributeType="Scalar" Center="Cell"><DataItem DataType="Int" Dimensions="428" Format="HDF" Precision="8">cylinder.h5:/data3</DataItem></Attribute><Attribute Name="gmsh:geometrical" AttributeType="Scalar" Center="Cell"><DataItem DataType="Int" Dimensions="428" Format="HDF" Precision="8">cylinder.h5:/data4</DataItem></Attribute></Grid></Domain></Xdmf>
```

Comment:
do we have to store all the binaries in the repo, or could those be generated on the fly?

Reply:
they dont need to be stored longterm, this is just a "byproduct", therefore also not found in the doit dependencies and targets. this is due to the gmsh mesh generation and transformation to fenics mesh.
it is overridden when run twice. we could make sure to delete it afterwards to keep everything "clean"

Reply:
Could we then just add that in the .gitignore file?

Reply:
Done!